### PR TITLE
Issue #2528: Theme the maintenance page in Basis.

### LINKED ISSUE #2528: Basis: Design and theme the Maintenance page
Current Basis
Screen Shot 2019-04-13 at 8 59 48 PM

Bartik:
backdrop-maintenance_mode-bartik

Seven:
backdrop-maintenance_mode-seven

Previous Basis:
backdrop-maintenance_mode-basis

---
PR: https://github.com/backdrop/backdrop/pull/2511
Milestone candidate tag for ~~1.12.3~~ meanwhile 1.12.7.

### PR DESCRIPTION
**Pull request by jenlampton**
Fixes https://github.com/backdrop/backdrop-issues/issues/2528

### CODE REVIEW

**Review by klonos — APPROVED** [review]
LGTM

**Review by olafgrabienski — COMMENTED** [review]
Easy change, great result!

However, Basis uses `rem` instead of `px` for padding, so far without exceptions, would be good to stick with it.

File: core/themes/basis/template.php [hunk 1]
```diff
@@ -27,6 +27,16 @@ function basis_preprocess_page(&$variables) {
   }
 }
 
+/**
+ * Prepares variables for page templates.
```
Comment (quicksketch) [inline core/themes/basis/template.php]:
```suggestion
 * Prepares variables for the maintenance page template.
```

**Comment by quicksketch** [comment]
Super thanks. Merged into 1.x and 1.12.x.

**Pull request merged**